tests/mamba/triton_reference/selective_state_update.py

```
@@ -9,25 +9,21 @@
 import torch
 import triton
 import triton.language as tl
-from packaging import version
 
-PAD_SLOT_ID = -1
-
-TRITON3 = version.parse(triton.__version__) >= version.parse("3.0.0")
-
-if TRITON3:
+try:
+    from .softplus import softplus  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)